seqr/views/apis/staff_api.py

```diff
@@ -409,6 +409,29 @@ def discovery_sheet(request, project_guid):
     })
 
 
+@staff_member_required(login_url=API_LOGIN_REQUIRED_URL)
+def success_story(request, success_story_types):
+
+    if success_story_types == 'all':
+        families = Family.objects.filter(~Q(success_story=None))
```

Comment:
A better way to do this is using an `isnull` filter: https://docs.djangoproject.com/en/1.11/ref/models/querysets/#std:fieldlookup-isnull